sys/linux/devio.txt

```diff
@@ -0,0 +1,129 @@
+# Copyright 2019 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# https://elixir.bootlin.com/linux/latest/source/drivers/usb/core/devio.c
+
+include <linux/ioctl.h>
+include <linux/usbdevice_fs.h>
+include <uapi/linux/fcntl.h>
+
+define MAXDRIVERNAME	USBDEVFS_MAXDRIVERNAME + 1
+
+resource fd_devio[fd]
+
+syz_open_dev$usb(dev ptr[in, string["/dev/bus/usb/00#/00#"]], id intptr, flags flags[open_flags]) fd_devio
+
+read$devio(fd fd_devio, buf buffer[out], count len[buf])
+
+mmap$devio(addr vma, len len[addr], prot flags[mmap_prot], flags flags[mmap_flags], fd fd_devio, offset fileoff)
+_ = __NR_mmap2
+
+ioctl$USBDEVFS_REAPURB(fd fd_devio, cmd const[USBDEVFS_REAPURB], arg ptr[out, int64])
```

Comment:
Add a comment that this returns a pointer to an URB in `arg`, I'll think what to do with this later.